src/main/java/it/multicoredev/carpetredcraftaddons/CarpetRedCraftSettings.java

```diff
@@ -188,6 +188,13 @@
     )
     public static boolean moreWanderingTrades = true;
 
+    @Rule(
+            desc="bonemealing a mossy block will became a moss_block.",
```

Comment:
`desc="Allow to obtain moss by bonemealing mossy blocks.",`

Cerchiamo di mantenere la stessa descrizione nella documentazione e nel gioco.